backend/mailman_lambda.py

```diff
@@ -26,6 +26,9 @@ def lambda_handler(event, context):
                 TableName='Registrations',
                 Key={'email': {'S': sender}},
             )
+            if not response:
```

Comment:
How is this different from what happens today? Wouldn't we want to just keep processing other emails if we fail on a single?

Reply:
Also wouldn't we want to do like validation so we drop requests if they dont have a subscription?